searx/static/themes/simple/.stylelintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "stylelint-config-standard"
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "declaration-empty-line-before": null,
+    "no-invalid-position-at-import-rule": null
+  }
```

Comment:
IMO this should be done in a separated commit, with a commit message why we want to activate:

- https://stylelint.io/user-guide/rules/list/declaration-empty-line-before/
- https://stylelint.io/user-guide/rules/list/no-invalid-position-at-import-rule/

Reply:
With `declaration-empty-line-before` enabled:
```less
#results.image-detail-open article.result-images[data-vim-selected] .detail {
  display: flex;
  flex-direction: column;
  position: fixed;
  .ltr-left(60rem);
  .ltr-right(0);
  top: @search-height;
```

has to be written
```less
#results.image-detail-open article.result-images[data-vim-selected] .detail {
  display: flex;
  flex-direction: column;
  position: fixed;

  .ltr-left(60rem);

  .ltr-right(0);
  top: @search-height;
```

---

With `no-invalid-position-at-import-rule` disabled, it allows to have these lines after the mixin declarations:
https://github.com/searxng/searxng/blob/739f0355f9f0120791f6f59ed0c5499097cf20bf/searx/static/themes/simple/src/less/style-rtl.less#L57

https://github.com/searxng/searxng/blob/739f0355f9f0120791f6f59ed0c5499097cf20bf/searx/static/themes/simple/src/less/style-rtl.less#L57

I can create a commit for this change.

Reply:
> I can create a commit for this change.

this would be cool / thanks a lot!

Reply:
Done.
I've merged the other commits (I have a backup if for reason it was not a good choice).